src/_includes/banner.html

```diff
@@ -2,8 +2,9 @@
 {% comment %}Drop the div .alert classes to revert to the default banner styling.{% endcomment -%}
 <div class="banner alert alert-info">
   <p class="banner__text">
-    <a href="/dart-2"><b>Dart 2.0</b></a> is now available!
-    <a href="/install">Install Dart</a>.
+    <a href="https://medium.com/dartlang/dart-2-stable-and-the-dart-web-platform-3775d5f8eac7" target="_blank">Dart 2.0 is now stable!</a>
+    <a href="/dart-2#migration">Migrate your code</a> or
+    <a href="/install">install Dart</a>.
     {% comment %}
```

Comment:
We can probably drop this comment now that are mentioning migration once again in the banner.